lib/timeline/component/CustomTime.js

```diff
@@ -187,13 +187,22 @@ class CustomTime extends Component {
 
   /**
    * Set custom marker.
-   * @param {string} title
+   * @param {string} [title] Title of the custom marker
+   * @param {boolean} [editable] Make the custom marker editable.
    */
-  setCustomMarker(title) {
+  setCustomMarker(title, editable) {
     const marker = document.createElement('div');
     marker.className = `vis-custom-time-marker`;
     marker.innerHTML = title;
     marker.style.position = 'absolute';
+    if (editable) {
+      marker.setAttribute('contenteditable', 'true');
+      marker.addEventListener('pointerdown', function () {
```

Comment:
Doing `focus()` is necessary since the default is prevented by [here](https://github.com/visjs/vis-timeline/blob/master/lib/module/hammer.js#L33).
If you any better idea, please share it.